src/trio/_core/_run.py

```diff
@@ -2244,6 +2254,13 @@ def run(
           propagates it.
 
     """
+    if strict_exception_groups is False:
+        warn_deprecated(
+            "trio.run(..., strict_exception_groups=False)",
+            version="0.24.1",
+            issue=2929,
+            instead="the default value of True and rewrite exception handlers to handle ExceptionGroups",
```

Comment:
Other messages begin "Use", otherwise identical.  copy/paste error?

Reply:
looking at `warn_deprecated`
```python
        msg += f"; use {_stringify(instead)} instead"
```
I might have noticed that supplies a `use` and only removed it in one of the three places. Fixed